WalletWasabi.Fluent/ViewModels/Dialog/IDialogHost.cs

```diff
@@ -0,0 +1,26 @@
+#nullable enable
```

Comment:
IMO we should just enable it for the whole project.

Reply:
yeah, let's enable it on another PR perhaps

Reply:
Did that in #4485